technical-doc/api.rst

```diff
@@ -5,114 +5,155 @@
+Massa JSON-RPC API is splitted in two parts:
 
 - **Private API**: used for node management. Default port: 33034 e.g. http://localhost:33034
 - **Public API**: used for blockchain interactions. Default port: 33035 e.g. http://localhost:33035
 
+Find the complete Massa `OpenRPC <https://spec.open-rpc.org/>`_ specification `here
+<https://raw.githubusercontent.com/massalabs/massa/main/massa-node/base_config/openrpc.json>`_.
+
+.. note::
+
+   We provide an `Interactive API Specification <https://playground.open-rpc.org/?schemaUrl=https://test.massa.net/api/v2&uiSchema[appBar][ui:input]=false&uiSchema[appBar][ui:inputPlaceholder]=Enter%20Massa%20JSON-RPC%20server%20URL&uiSchema[appBar][ui:logoUrl]=https://massa.net/favicons/favicon.ico&uiSchema[appBar][ui:splitView]=false&uiSchema[appBar][ui:darkMode]=false&uiSchema[appBar][ui:title]=Massa&uiSchema[appBar][ui:examplesDropdown]=false&uiSchema[methods][ui:defaultExpanded]=false&uiSchema[methods][ui:methodPlugins]=true&uiSchema[params][ui:defaultExpanded]=false>`_ which documents every available method and allows you to play with the API.
+
 .. warning::
 
     Massa has a new experimental API with both Http and WebSocket support: Default port: 33036
 
     - **Http**: used for node management and blockchain interactions. e.g. http://localhost:33036
     - **WebSocket**: used for streaming blockchain events. e.g. ws://localhost:33036
 
-Find the complete Massa `OpenRPC <https://spec.open-rpc.org/>`_ specification `here
-<https://raw.githubusercontent.com/massalabs/massa/main/massa-node/base_config/openrpc.json>`_.
-
 Integrations
 ------------
 
-- **JavaScript**: use `massa-web3.js <https://github.com/massalabs/massa-web3>`_.
-- **Playground**: use `Massa Playground
-  <https://playground.open-rpc.org/?schemaUrl=https://test.massa.net/api/
-  v2&uiSchema[appBar][ui:input]=false&uiSchema[appBar][ui:inputPlaceholder]
-  =Enter%20Massa%20JSON-RPC%20server%20URL&uiSchema[appBar][ui:logoUrl]=
-  https://massa.net/favicons/favicon.ico&uiSchema[appBar][ui:splitView]=
-  false&uiSchema[appBar][ui:darkMode]=false&uiSchema[appBar][ui:title]=
-  Massa&uiSchema[appBar][ui:examplesDropdown]=false&uiSchema[methods]
-  [ui:defaultExpanded]=false&uiSchema[methods][ui:methodPlugins]=true&uiSchema
-  [params][ui:defaultExpanded]=false>`_.
-- **Postman**: use `Postman collection <https://github.com/massalabs/massa-docs/tree/main/postman>`_.
+**JavaScript**: see `massa-web3.js <https://github.com/massalabs/massa-web3>`_.
+
+**Postman**: is a tool for software developers to create, test and debug API requests. You can create collections of requests in Postman workspaces for easy collaboration on projects. 
+For more information, please visit `Postman learning center <https://learning.postman.com/docs/getting-started/introduction/>`_.
+
+Find all maintained Massa Postman collections in our official `workspace <https://www.postman.com/massalabs>`_.
+
+
+Using Massa Offical Node vs Running Your Own Node on Testnet
```

Comment:
I would simply provide a table like what is done here: https://chainlist.org/chain/1
We can also add some community RPCs as they are developped.

Reply:
This is a good idea, I talked about it with tech ambassadors.

Reply:
![image](https://user-images.githubusercontent.com/22281426/223692222-dd02ab6b-a73a-4f26-beca-fe0514202949.png)

features: JsonRPC, gRPC,WebSocket

Reply:
I kept the Postman Workspace which provided a real time documentation of all our stable and experimental features.

I added a table with Massa official node, and launched a conversation with tech ambassadors about adding community nodes to this section later